qemu/qmp.go

```diff
@@ -1033,6 +1052,30 @@ func (q *QMP) ExecQueryMemoryDevices(ctx context.Context) ([]MemoryDevices, erro
 	return memoryDevices, nil
 }
 
+// ExecQueryCpus returns a slice with the list of `CpuInfo`
+// Since qemu 2.12, we have `query-cpus-fast` as a better choice in production
+// we can still choose `ExecQueryCpus` for compatibility though not recommended.
```

Comment:
so you don't recommend to use this function `ExecQueryCpus`?

Reply:
According the QMP documentation, YES.
But `query-cpus-fast` requires so high qemu version and the return value is so different, so there's no way to automatically choose best one for end user. I have to use the old compatible one.

Reply:
jfyi in kata we know what version of qemu the user has

Reply:
Oh, yes, I think you're right!